[2022-01-29 14:07:45,370] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2020-02-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:45,417] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2020-02-02T06:00:00+00:00 [queued]>
[2022-01-29 14:07:45,418] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:45,418] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-29 14:07:45,418] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:07:45,463] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2020-02-02 06:00:00+00:00
[2022-01-29 14:07:45,469] {standard_task_runner.py:52} INFO - Started process 352 to run task
[2022-01-29 14:07:45,474] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2020-02-02T06:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmpyjpqxxb7', '--error-file', '/tmp/tmpzyicv742']
[2022-01-29 14:07:45,475] {standard_task_runner.py:77} INFO - Job 31: Subtask get_data
[2022-01-29 14:07:45,642] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2020-02-02T06:00:00+00:00 [running]> on host d1e1fe005577
[2022-01-29 14:07:45,835] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2020-02-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-02-02T06:00:00+00:00
[2022-01-29 14:07:45,838] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-29 14:07:45,840] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "helloworld!!"']
[2022-01-29 14:07:45,855] {subprocess.py:85} INFO - Output:
[2022-01-29 14:07:45,859] {subprocess.py:89} INFO - helloworld!!
[2022-01-29 14:07:45,860] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-29 14:07:45,936] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20200202T060000, start_date=20220129T140745, end_date=20220129T140745
[2022-01-29 14:07:46,025] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 14:07:46,133] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
